base/random.jl

```diff
@@ -272,11 +305,13 @@ Pick a random element or array of random elements from the set of values specifi
 
 # Examples
 
-```julia-repl
-julia> rand(Int, 2)
```

Comment:
It's nice to have an example without an explicit RNG. If you want it reproducible, I suggest you call `srand(1234)` instead of creating `rng`.